tests/unit/AmalgamationFiling.spec.ts

```diff
@@ -16,12 +16,13 @@ const vuetify = new Vuetify({})
 setActivePinia(createPinia())
 const businessStore = useBusinessStore()
 
-describe('Amalgamation Filing', () => {
+describe('Regular amalgamation Filing', () => {
   let wrapper: Wrapper<AmalgamationFiling>
 
   beforeAll(() => {
     // init store
     businessStore.setLegalName('MY COMPANY')
+    sessionStorage.setItem('TEMP_REG_NUMBER', null)
```

Comment:
Why is this needed?

Conversely, why is setting BUSINESS_ID in session storage not needed?

Reply:
Somehow, the test was not able to proceed because it was failing at `v-if="!!tempRegNumber && isStatusCompleted"`.
I tried to set few attributes but only this worked.

![image](https://github.com/bcgov/business-filings-ui/assets/116035339/15529e0b-dc64-4bb8-993a-e9050a57ea87)

Can you suggest what else I could try and it might work?

Reply:
Well, I guess the emulated session storage in the tests contained left-over data for that value.

There are 2 approaches to having expected data in store/session:
1. at the end of other tests, clean up
2. at the beginning of your test, initialize as needed

I think option 2 is preferable since it doesn't depend on the other tests. So your solution is fine (but you could add a code comment like, "ensure tempRegNumber doesn't exist").